e2e/playwright/snapshot-tests.spec.ts

```diff
@@ -274,24 +250,55 @@ const part001 = startSketchOn('-XZ')
       presentation: 'pretty',
     })
   )
-
-  // TODO: gltfs don't seem to work with snap shots. push onto exportLocations once it's figured out
-  await doExport({
-    type: 'gltf',
-    storage: 'standard',
-    presentation: 'pretty',
-  })
+  exportLocations.push(
+    await doExport({
+      type: 'gltf',
+      storage: 'standard',
+      presentation: 'pretty',
+    })
+  )
 
   // close page to disconnect websocket since we can only have one open atm
   await page.close()
 
   // snapshot exports, good compromise to capture that exports are healthy without getting bogged down in "did the formatting change" changes
   // context: https://github.com/KittyCAD/modeling-app/issues/1222
-  for (const { modelPath, imagePath, outputType } of exportLocations) {
-    console.log(
-      `taking snapshot of using: "zoo file snapshot --output-format=png --src-format=${outputType} ${modelPath} ${imagePath}"`
-    )
-    const cliCommand = `export ZOO_TOKEN=${secrets.snapshottoken} && zoo file snapshot --output-format=png --src-format=${outputType} ${modelPath} ${imagePath}`
+  for (let { modelPath, imagePath, outputType } of exportLocations) {
+    // May change depending on the file being dealt with
+    let cliCommand = `export ZOO_TOKEN=${secrets.snapshottoken} && zoo file snapshot --output-format=png --src-format=${outputType} ${modelPath} ${imagePath}`
+
+    // This is actually a zip file.
+    if (modelPath.includes('gltf-standard.gltf')) {
+      console.log('Extracting files from archive')
+      const readZipFile = fsp.readFile(modelPath)
+      const unzip = (archive: any) =>
+        Object.values(archive.files).map((file: any) => ({
+          name: file.name,
+          promise: file.async('nodebuffer'),
+        }))
+      const writeFiles = (files: any) =>
+        Promise.all(
+          files.map((file: any) =>
+            file.promise.then((data: any) => {
+              console.log(`Writing ${file.name}`)
+              return fsp.writeFile(file.name, data).then(() => file.name)
```

Comment:
Un tested by maybe this should be something like
```suggestion
              return fsp.writeFile(`./e2e/playwright/export-snapshots/${file.name}`, data).then(() => file.name)
```
So that these file end up in export-snapshots and therefore gitignored along with all the other exports?

Reply:
Ah is that the condition? Sounds good then -  let's see...

Reply:
(I'll use `path` to properly compute that path)